### ISSUE TITLE
all_visitors fixture is working incorrectly

### ISSUE THREAD

**Issue by valignatev** [wemake-services/wemake-python-styleguide#258]
## What's wrong

Currently, `all_visitors` fixture contains some entries twice because they're imported from both visitors and presets subdirs with different ids. It means that `test_all_visitors_contained_in_checker` currently works only accidentally.

## How is that should be
Ideally, `all_visitors` shouldn't import visitors from presets subdir, because those go directly into Checker, but instead only look at visitors modules directly.  

I'll provide PR for this issue, hopefully it'll make more sense together

UPD: proof, return value of the fixture, without set curly parens and sorted so you can clearly see duplicates:
```python
<class 'wemake_python_styleguide.visitors.ast.classes.WrongClassVisitor'>
<class 'wemake_python_styleguide.visitors.ast.classes.WrongClassVisitor'>
<class 'wemake_python_styleguide.visitors.ast.comparisons.ComparisonSanityVisitor'>
<class 'wemake_python_styleguide.visitors.ast.comparisons.WrongComparisionOrderVisitor'>
<class 'wemake_python_styleguide.visitors.ast.functions.WrongFunctionCallVisitor'>
<class 'wemake_python_styleguide.visitors.ast.functions.WrongFunctionCallVisitor'>
<class 'wemake_python_styleguide.visitors.ast.imports.WrongImportVisitor'>
<class 'wemake_python_styleguide.visitors.ast.imports.WrongImportVisitor'>
<class 'wemake_python_styleguide.visitors.ast.keywords.WrongForElseVisitor'>
<class 'wemake_python_styleguide.visitors.ast.keywords.WrongKeywordVisitor'>
<class 'wemake_python_styleguide.visitors.ast.keywords.WrongListComprehensionVisitor'>
<class 'wemake_python_styleguide.visitors.ast.keywords.WrongRaiseVisitor'>
<class 'wemake_python_styleguide.visitors.ast.keywords.WrongTryFinallyVisitor'>
<class 'wemake_python_styleguide.visitors.ast.modules.EmptyModuleContentsVisitor'>
<class 'wemake_python_styleguide.visitors.ast.modules.EmptyModuleContentsVisitor'>
<class 'wemake_python_styleguide.visitors.ast.naming.WrongModuleMetadataVisitor'>
<class 'wemake_python_styleguide.visitors.ast.naming.WrongNameVisitor'>
<class 'wemake_python_styleguide.visitors.ast.numbers.MagicNumberVisitor'>
<class 'wemake_python_styleguide.visitors.ast.strings.WrongStringVisitor'>
<class 'wemake_python_styleguide.visitors.ast.strings.WrongStringVisitor'>
<class 'wemake_python_styleguide.visitors.filenames.wrong_module_name.WrongModuleNameVisitor'>
<class 'wemake_python_styleguide.visitors.filenames.wrong_module_name.WrongModuleNameVisitor'>
<class 'wemake_python_styleguide.visitors.tokenize.comments.WrongCommentVisitor'>
<class 'wemake_python_styleguide.visitors.tokenize.comments.WrongCommentVisitor'>
<class 'wemake_python_styleguide.visitors.tokenize.primitives.WrongPrimitivesVisitor'>
<class 'wemake_python_styleguide.visitors.tokenize.primitives.WrongPrimitivesVisitor'>
```
